app/src/main/java/com/kuit/kuit6android/ui/home/component/GoOrderItem.kt

```diff
@@ -0,0 +1,73 @@
+package com.kuit.kuit6android.ui.home.component
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Button
+import androidx.compose.material3.ButtonDefaults
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+
+@Composable
+fun GoOrderItem(modifier: Modifier = Modifier) {
+    Box(
+        modifier = modifier
+            .background(Color(0xFF0C2F65), shape = RoundedCornerShape(size = 8.dp))
+            .size(width = 306.dp, height = 77.dp)
+            .padding(all = 12.dp),
```

Comment:
```suggestion
        modifier = modifier
            .fillMaxWidth()
            .padding(horizontal = 27.dp)
            .background(Color(0xFF0C2F65), shape = RoundedCornerShape(size = 8.dp))
            .padding(all = 12.dp),
```

이 부분도 사이즈를 고정하기보다는 외부 패딩과 내부 패딩만 설정하고, 화면 사이즈에 맞게 표시되도록 fillMaxWidth를 사용하는 게 더 좋을 것 같습니다